### PR TITLE
LTD-2458: Only allow specified users to change config rules

### PR DESCRIPTION
**Pull request by saruniitr**
## Change description

Config rules include Routing rules and flagging rules, we don't want
everyone (even if they are Admin) to change these rules so only
allow specified list of users.

### CODE REVIEW

**Review by depsiatwal — APPROVED** [review]
LGTM

File: unit_tests/caseworker/flags/test_views.py [hunk 1]
```diff
@@ -12,10 +12,20 @@ def setup(
...
 
 
+def test_view_flagging_rules(authorized_client):
+
+    url = reverse("flags:flagging_rules")
+    response = authorized_client.get(url)
+    assert response.status_code == 200
+    assert response.context["can_change_config"] == False
+
+
```
Comment (kevincarrogan) [inline unit_tests/caseworker/flags/test_views.py:28]: Can you add the test that asserts the `can_change_config == True` case and then this is all good.

**Comment by depsiatwal** [comment]
did we have any e2e tests around this

**Review by kevincarrogan — APPROVED** [review]

File: unit_tests/caseworker/flags/test_views.py (hunk 1 above)
Comment (saruniitr) [reply unit_tests/caseworker/flags/test_views.py:28]: :+1:

**Pull request merged**